src/app/components/common-timetable/timetable/timetable.component.ts

```diff
@@ -0,0 +1,116 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Week} from "../../../models/Week";
+import {TrimDayName} from "../../../utils/TrimDayName";
+import {Day} from "../../../models/Day";
+
+@Component({
+  selector: 'app-timetable',
+  templateUrl: './timetable.component.html',
+  styleUrls: ['./timetable.component.scss']
+})
+export class TimetableComponent implements OnInit {
+  @Input()
+  mfzWeeks: Week[] | [] = []
+
+  @Input()
+  nurWeeks: Week[] | [] = []
+
+  monthsName = [
```

Comment:
should be a const it is not changable or make it enum

Reply:
made it enum